components/admin_console/billing/company_info_edit.tsx

```diff
@@ -28,16 +197,64 @@ const CompanyInfoEdit: React.FC<Props> = () => {
             </div>
             <div className='admin-console__wrapper'>
                 <div className='admin-console__content'>
-                    <div style={{border: '1px solid #000', width: '100%', height: '432px'}}>
-                        {'Edit Company Info View'}
+                    <div className='CompanyInfoEdit__card'>
+                        <div className='CompanyInfoEdit__form'>
+                            <div className='section-title'>
+                                <FormattedMessage
+                                    id='admin.billing.company_info_edit.companyDetails'
+                                    defaultMessage='Company Details'
+                                />
+                            </div>
+                            <div className='form-row'>
+                                <Input
+                                    name='companyName'
+                                    type='text'
+                                    value={companyName}
+                                    onChange={updateState(setCompanyName)}
+                                    placeholder={Utils.localizeMessage('admin.billing.company_info.companyName', 'Company name')}
+                                    required={true}
+                                />
+                            </div>
+                            <div className='form-row'>
+                                <Input
```

Comment:
@matthewbirtch Should we hide the `numEmployees` tag under the name until they have added something? Doesn't seem to provide any value showing `0 employees` there from the start

Reply:
Makes sense to me.

Reply:
yep, makes sense to me too

Reply:
@devinbinnie could we include as part of this PR?